### PR TITLE
[kube-metrics] Add metrics to kube

### PR DESCRIPTION
**Pull request by Lewiky**
Who loves reviewing yaml!!! I do !!!

Adds prometheus and grafana to el kubernooties

## Test plan
* `source deploy.sh`

* `minikube service grafana`

* make some requests as usual and watch them come in the grafana dash

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -14,10 +14,29 @@ minikube start --vm-driver=virtualbox
...
 
+# Prometheus
+kubectl create configmap prometheus-file-config --from-file "$BASEDIR/prometheus/prometheus.yml" -o=yaml
+
+# Grafana
+kubectl create configmap grafana-datasource-config --from-file "$BASEDIR/grafana/provisioning/datasources/datasource.yml" -o=yaml
+
+for file in "$BASEDIR/grafana/provisioning/dashboards/"*
+do
+  filename="${file##*/}" # Get everything after the final /
```
Comment (jaylees14) [inline deploy.sh]: Using bash substrings, you brave boy 😉  I think you might need `/bin/bash` at the top for this always to be correct.

Comment (jaylees14) [reply deploy.sh]: Also.. what happens if there's a folder in this directory?

**Review by jaylees14 — COMMENTED** [review]
Changes work locally for me, just a small comment in the shell script

**Review by c-harding — APPROVED** [review]
There‘s witchcraft, but it’s documented, so sure

**Review by jaylees14 — APPROVED** [review]
LGTM

**Pull request merged**